[2024-02-08T00:13:46.472+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lighthouse_challenge.extracts.teste manual__2024-02-08T00:13:43.256294+00:00 [queued]>
[2024-02-08T00:13:46.479+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lighthouse_challenge.extracts.teste manual__2024-02-08T00:13:43.256294+00:00 [queued]>
[2024-02-08T00:13:46.479+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-08T00:13:46.492+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extracts.teste> on 2024-02-08 00:13:43.256294+00:00
[2024-02-08T00:13:46.503+0000] {standard_task_runner.py:60} INFO - Started process 93 to run task
[2024-02-08T00:13:46.509+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'lighthouse_challenge', 'extracts.teste', 'manual__2024-02-08T00:13:43.256294+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpniymr138']
[2024-02-08T00:13:46.511+0000] {standard_task_runner.py:88} INFO - Job 34: Subtask extracts.teste
[2024-02-08T00:13:46.555+0000] {task_command.py:423} INFO - Running <TaskInstance: lighthouse_challenge.extracts.teste manual__2024-02-08T00:13:43.256294+00:00 [running]> on host 793d4bc8add3
[2024-02-08T00:13:46.628+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lighthouse_challenge' AIRFLOW_CTX_TASK_ID='extracts.teste' AIRFLOW_CTX_EXECUTION_DATE='2024-02-08T00:13:43.256294+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-08T00:13:43.256294+00:00'
[2024-02-08T00:13:46.633+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-08T00:13:46.634+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2024-02-08T00:13:46.644+0000] {subprocess.py:86} INFO - Output:
[2024-02-08T00:13:46.650+0000] {subprocess.py:93} INFO - /tmp/***tmpbz3m5gj5
[2024-02-08T00:13:46.651+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-08T00:13:46.667+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=lighthouse_challenge, task_id=extracts.teste, execution_date=20240208T001343, start_date=20240208T001346, end_date=20240208T001346
[2024-02-08T00:13:46.730+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-08T00:13:46.768+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
